azurerm/resource_arm_application_gateway.go

```diff
@@ -2657,11 +2703,42 @@ func flattenApplicationGatewayWafConfig(input *network.ApplicationGatewayWebAppl
 		output["max_request_body_size_kb"] = int(*input.MaxRequestBodySizeInKb)
 	}
 
+	if input.DisabledRuleGroups != nil {
+		output["disabled_rule_group"] = flattenApplicationGatewayDisabledRuleGroups(input.DisabledRuleGroups)
+	}
+
 	results = append(results, output)
 
 	return results
 }
 
+func flattenApplicationGatewayDisabledRuleGroups(input *[]network.ApplicationGatewayFirewallDisabledRuleGroup) []interface{} {
+	result := make([]interface{}, 0)
+	if input == nil {
+		return []interface{}{}
+	}
+
+	for _, v := range *input {
+		output := map[string]interface{}{}
+
+		output["rule_group_name"] = string(*v.RuleGroupName)
+
+		if v.Rules != nil {
+			for _, r := range *v.Rules {
+				rulesOutput := map[string]interface{}{}
+
+				rulesOutput["rules"] = r
+
+				result = append(result, rulesOutput)
```

Comment:
Shoudn't the rulesOutput be appended to the output instead of result?

Reply:
Thanks for your help!